resources/js/src/app/components/pageDesign/MobileNavigation.vue

```diff
@@ -20,7 +21,7 @@
                 </li>
 
                 <li class="ddown" v-for="category in dataContainer1.categories">
-                    <a :href="getCategoryUrl(category.url)">{{ category.details[0].name }}</a>
+                    <a :href="getCategoryUrl(category.url)" itemprop="name" role="menuitem">{{ category.details[0].name }}</a>
```

Comment:
If we change the role to navigation we don't need to set the role to the child anymore, as links with navigation have the implicit roles of "link" which is correct semantic role for navigation items.

```suggestion
                    <a :href="getCategoryUrl(category.url)" itemprop="name">{{ category.details[0].name }}</a>
```